### ISSUE TITLE
Mobile A11y Testing 11/1

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-toggle-button#63]
On Android with Talkback, when I double tap to toggle, I get no spoken feedback of the new state. I have to move focus and then place focus back on that same item to hear the new state. We need some sort of spoken alert that the state has changed.

**Comment by notwaldorf** [comment]
I think the same happens on Desktop too, as well for checkboxes and radio buttons (change is not announced). Can you confirm that a native `<input type="checkbox">` announces a change in state?
